website/docs/r/compute_https_health_check.html.markdown

```diff
@@ -10,11 +10,13 @@ description: |-
 
 Manages an HTTPS health check within GCE. This is used to monitor instances
 behind load balancers. Timeouts or HTTPS errors cause the instance to be
-removed from the pool. For more information, see [the official
+removed from the pool. Creates an HttpHealthChecks resource. For more information, see [the official
```

Comment:
Same here, this addition doesn't really clarify anything for me, and is more confusing than anything.

Reply:
Fixed.